### PR TITLE
Create gRPC objects using managed object service

### PR DESCRIPTION
**Pull request by wtlucy**
for #13229

### CODE REVIEW

**Comment by wtlucy** [comment]
#build #spawn.fullfat.buckets=com.ibm.ws.grpc_fat

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
I've done a rebase, let's try the tests again: #build #spawn.fullfat.buckets=com.ibm.ws.grpc_fat

**Comment by wtlucy** [comment]
#run-libby-bot

File: dev/com.ibm.ws.grpc_fat/test-applications/HelloWorldService.war/src/com/ibm/ws/grpc/fat/helloworld/service/HelloWorldServlet.java [hunk 1]
```diff
@@ -35,13 +36,31 @@ public class HelloWorldServlet extends HttpServlet {
     // implementation of the "GreeterGrpc" service
     public static final class GreeterImpl extends GreeterGrpc.GreeterImplBase {
 
+        @Inject
```
Comment (hutchig) [inline dev/com.ibm.ws.grpc_fat/test-applications/HelloWorldService.war/src/com/ibm/ws/grpc/fat/helloworld/service/HelloWorldServlet.java:39]: Link

**Pull request merged**